commons/src/main/java/io/hyscale/commons/logger/WorkflowLogger.java

```diff
@@ -217,4 +217,17 @@ public static void logPersistedActivities() {
         }
         persistedActivities.clear();
     }
+    
+    
+    public static void persistError(Activity activity, String... args) {
```

Comment:
If we have by tag, why do we need this separately ?

Reply:
Added as a helper method, that can be called directly. Should we removed this?